packages/gui/src/components/nfts/NFTPreview.tsx

```diff
@@ -47,8 +293,18 @@ export type NFTPreviewProps = {
   elevate?: boolean;
   background?: any;
   hideStatusBar?: boolean;
```

Comment:
Missing prop definition for `isCompact`

```suggestion
  hideStatusBar?: boolean;
  isCompact?: boolean;
```

Reply:
Done.